.changeset/rotten-steaks-impress.md

```diff
@@ -0,0 +1,5 @@
+---
+"@effect/build-utils": minor
+---
+
+Use `sideEffects: []` instead of `sideEffects: false`
```

Comment:
out of curiosity - what prompted this change? I don't recall specifics at all but I distinctly remember that `false` has a better support across tools than an array variant (at least at some point in time)

Reply:
This was the original discussion (explicit): https://discord.com/channels/795981131316985866/1174418279276757002/1174680689971564574 :-P